bin/skonfig

```diff
@@ -1,23 +1,36 @@
 #!/usr/bin/env python3
 
 import os
+import subprocess
 import sys
 import traceback
 
-sys.path.insert(
-    0,
-    os.path.realpath(
-        os.path.join(
-            os.path.dirname(os.path.realpath(__file__)),
-            os.pardir
-        )
-    ),
+skonfig_bindir_pardir = os.path.realpath(
```

Comment:
Instead of `realpath` + `pardir`, just wrap another `dirname`…

```python3
skonfig_bindir_pardir = os.path.dirname(os.path.dirname(os.path.realpath(__file__)))